src/components/molecules/ChatPoll/ChatPoll.scss

```diff
@@ -46,6 +55,11 @@ button.ChatPoll__question {
   &__text {
     font-size: $font-size--lg;
     margin-bottom: $spacing--md;
+
+    // This direct element style is used with RenderMarkdown
+    p {
+      margin-bottom: 0;
+    }
```

Comment:
What aspect of `RenderMarkdown` is causing you to need to do this? (I know it wraps basic text in `<p></p>`, but what is breaking because of that?)

Reply:
@0xdevalias `RenderMarkdown` is rendering `p` by default which has default margin bottom. Please check: https://github.com/sparkletown/sparkle/pull/1806#discussion_r668676044